homeassistant/components/alexa/state_report.py

```diff
@@ -86,7 +86,9 @@ async def async_entity_state_listener(
             return
 
         if should_doorbell:
-            if new_state.state == STATE_ON:
+            if (
+                old_state is None or old_state.state != STATE_ON
+            ) and new_state.state == STATE_ON:
```

Comment:
Just was thinking, should we not first check the new state, if that is not on, than that is only one check.